examples/anthropic_thinking_example.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+"""
+Example demonstrating Claude's extended thinking feature with LiveKit Agents.
+
+This example shows how to enable and configure Claude's thinking capabilities,
+which allow Claude to show its internal reasoning process before delivering
+the final answer.
+
+Requirements:
+- ANTHROPIC_API_KEY environment variable set
+- A Claude model that supports extended thinking (claude-3-7-sonnet, claude-4-sonnet, claude-4-opus)
+"""
+
+import asyncio
+import logging
+import os
+
+from livekit.agents.llm import ChatContext
+from livekit.plugins import anthropic
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+async def main():
+    # Ensure we have an API key
+    if not os.environ.get("ANTHROPIC_API_KEY"):
+        logger.error("Please set the ANTHROPIC_API_KEY environment variable")
+        return
+
+    # Example 1: Using thinking with a dict configuration
+    print("=== Example 1: Basic Thinking Configuration ===")
+    llm_with_thinking = anthropic.LLM(
+        model="claude-3-7-sonnet-20250219",  # Make sure to use a model that supports thinking
+        thinking={
+            "type": "enabled",
+            "budget_tokens": 10000,  # Allocate 10k tokens for thinking
+        },
+    )
+
+    # Create a chat context with a complex problem
+    chat_ctx = ChatContext().append(
+        text="You are a helpful math tutor. Think step by step through problems."
+    )
```

Comment:
```suggestion
    chat_ctx = ChatContext.empty()
    chat_ctx.add_message(
        role="system",
        content="You are a helpful math tutor. Think step by step through problems.",
    )
```

same for other `ChatContext().append`, which is not a valid method.